addons/account/views/account_move_views.xml

```diff
@@ -1075,7 +1075,7 @@
                                                domain="[('partner_id', '=', bank_partner_id)]"
                                                attrs="{'readonly': [('state', '!=', 'draft')]}"/>
                                         <field name="qr_code_method"
-                                               attrs="{'invisible': [('display_qr_code', '=', False)]}"/>
+                                               attrs="{'invisible': ['|', ('display_qr_code', '=', False), ('move_type', '=', 'out_refund')]}"/>
```

Comment:
I'd rather have display_qr_code computed as falsy for out_refund. 

It has the benefit of 
* fixing the bug, without needing an update of the module
* staying consistent with the variable meaning